zerver/views/webhooks/trello/__init__.py

```diff
@@ -0,0 +1,30 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+import ujson
+from zerver.models import get_client
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+from .card_actions import SUPPORTED_CARD_ACTIONS, process_card_action
+from .exceptions import UnsupportedAction
+
+
+@api_key_only_webhook_view
+@has_request_variables
+def api_trello_webhook(request, user_profile, stream=REQ(default='trello')):
+    payload = ujson.loads(request.body)
```

Comment:
Should catch an exception if `request.body` isn't valid JSON, like stash.py does